doc/api/child_process.md

```diff
@@ -405,6 +405,8 @@ changes:
     `'/bin/sh'` on UNIX, and `process.env.ComSpec` on Windows. A different
     shell can be specified as a string. See [Shell Requirements][] and
     [Default Windows Shell][]. **Default:** `false` (no shell).
+  * `windowsVerbatimArguments` {Boolean} No quoting or escaping of arguments is
```

Comment:
This is supported on other child process methods. There are some additional details that should probably be included. It is enabled unconditionally when the `shell` option is `true` on Windows. This means it's also turned for for `exec()` for example.

Reply:
So, just to make sure, you want me to both:

1) Add the doc to the methods that call `spawn` under the hood (`exec` family as well as `fork` from the looks of it)
2) Mention that it is automatically set when `shell` is set to `true`

Reply:
> 1. Add the doc to the methods that call spawn under the hood (exec family as well as fork from the looks of it)

Yes. I'm not a huge fan of it, but we repeat the options in this file a lot.

> 2. Mention that it is automatically set when shell is set to true

Yes, but only on Windows. `exec()` and `execSync()` are a bit special because you can't disable `shell`, and thus cannot disable WVA.

Reply:
Ok, so I updated the docs for `fork`, `spawn`, and `spawnSync` since the `exec` family always enables `shell` (`fork` always disables `shell`, but `windowsVerbatimArguments` is still able to be overwritten).

Also added in that the default value for `windowsVerbatimArguments` is `false`.